### PR TITLE
updated tests to fit discussion_urls

### PR DESCRIPTION
**Pull request by Ashar2shahid**
One thing to keep in mind is that you can set the discussion url for a vote that does not exist. Maybe have a check in setDiscussionUrl to avoid doing that?

### CODE REVIEW

**Comment by bbenligiray** [comment]
> One thing to keep in mind is that you can set the discussion url for a vote that does not exist

The dashboard will use Convenience.sol to fetch the discussion URLs for existing votes so this is not an issue

File: packages/convenience/contracts/Convenience.sol [hunk 1]
```diff
@@ -38,8 +38,7 @@ contract Convenience is Ownable  {
     /// @dev The owner privileges here do not pose a serious security risk, the
     /// worst that can happen is that the discussion URL will malfunction
     /// @param votingAppType Enumerated voting app type (primary or secondary)
-    /// @param voteId Array of vote ID for which discussion URL will be
-    /// updated
+    /// @param voteId vote ID for which discussion URL will be updated
```
Comment (bbenligiray) [inline packages/convenience/contracts/Convenience.sol:41]: Nice, thanks

**Review by bbenligiray — APPROVED** [review]
Thanks, looks good. I will add events for the `onlyOwner` methods and add another test for this where the discussion URL is for a secondary proposal.

**Comment by Ashar2shahid** [comment]
> and add another test for this where the discussion URL is for a secondary proposal.

not quite sure by what you mean by this? do you mean when the VotingAppType is secondary?

**Comment by bbenligiray** [comment]
Yeah, referring to https://github.com/api3dao/api3-dao/blob/to-revision/packages/convenience/test/Convenience.sol.js#L186

**Pull request merged**